apis/v1beta1/config.go

```diff
@@ -398,21 +424,21 @@ type MetricsConfig struct {
 	//  - "basic" is the recommended and covers the basics of the service telemetry.
 	//  - "normal" adds some other indicators on top of basic.
 	//  - "detailed" adds dimensions and views to the previous levels.
-	Level string `json:"level,omitempty" yaml:"level,omitempty"`
+	Level string `json:"level,omitempty" yaml:"level,omitempty" mapstructure:"level,omitempty"`
```

Comment:
why is this `mapstructure` needed?

Reply:
We use `mapstructure`  to decode and merge the telemetry struct into `map[string]interface{}` and without `omitempty` keys with empty values will appear.

Reply:
I removed the usage of mapstructure.